packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -2,30 +2,89 @@
     <div
         :class="$style['c-rating']"
         data-test-id="rating-component">
-        {{ copy.text }}
+        <ul :class="$style['c-rating-star-wrapper']">
+            <li
+                v-for="star in maxStarRating"
+                :key="star"
+                :class="$style['c-rating-star']">
+                <star-filled-icon
+                    v-if="getRating(star)"
+                    :class="$style['c-rating-star-filled']" />
+                <star-icon
+                    v-else
+                    :class="$style['c-rating-star-empty']" />
+            </li>
+            <span
+                data-test-id="c-rating-description"
+                class="is-visuallyHidden">
+                {{ starRating }} {{ setRatingDescription }} {{ maxStarRating }}
+            </span>
+        </ul>
     </div>
 </template>
 
 <script>
+import {
+    StarFilledIcon,
+    StarIcon
+} from '@justeattakeaway/pie-icons-vue';
 import { globalisationServices } from '@justeat/f-services';
 import tenantConfigs from '../tenants';
 
 export default {
     name: 'VRating',
-    components: {},
+    components: {
+        StarFilledIcon,
+        StarIcon
+    },
     props: {
         locale: {
             type: String,
             default: ''
+        },
+        starRating: {
+            type: Number,
+            required: true
+        },
+        maxStarRating: {
+            type: Number,
+            default: 5
         }
     },
+
     data () {
         const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
         const localeConfig = tenantConfigs[locale];
 
         return {
-            copy: { ...localeConfig }
+            copy: { ...localeConfig },
+            rating: this.starRating
         };
+    },
+
+    computed: {
+        /**
+         * Return singular description if one star.
+         *
+         * @returns {string|*}
+         */
+        setRatingDescription () {
```

Comment:
I think as per my other comment we can use i18n pluralization here. However just to be nitpicky I'm not sure if `set` is the correct verb to use in this method. Feels more like either a `get` or just `ratingDescription` to me - purely my opinion though!